cognite/client/_api/assets.py

```diff
@@ -285,7 +285,7 @@ def retrieve_subtree(self, id: int = None, external_id: str = None, depth: int =
         utils._auxiliary.assert_exactly_one_of_id_or_external_id(id, external_id)
```

Comment:
Update the return statement on the docstring. It is no longer sorted topologically.